librz/core/cmd_api.c

```diff
@@ -79,6 +79,38 @@ RZ_IPI int rz_output_mode_to_char(RzOutputMode mode) {
 	return -1;
 }
 
+RZ_IPI RzOutputMode rz_char_to_output_mode(char input) {
```

Comment:
Actually, I think you can rename `suffix2mode` to `rz_char_to_output_mode` and use that instead of this code.

Reply:
Alright, I'll do that.

Reply:
I have also modified `suffix2mode()` so that it compares a character instead of a substring. I hope that's fine.